container-registry/container-analysis/pom.xml

```diff
@@ -0,0 +1,83 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.containerregistry</groupId>
+  <artifactId>containeranalysis</artifactId>
+  <version>1.0</version>
+  <packaging>jar</packaging>
+
+  <name>containeranalysis</name>
+  <url>http://maven.apache.org</url>
+
+  <!-- local maven repo contains container analysis client library -->
```

Comment:
How are you planning to release this?

Reply:
The library is currently [publicly hosted for download on GCS](https://cloud-dot-devsite.googleplex.com/container-registry/docs/vulnerability-scanning), and it's on track to be added to the maven repo once it reaches beta

Reply:
Then why have you included it in the repo?

Reply:
The version currently hosted on GCS is just a .tar.gz of a gradle project, so I needed some way to integrate it into the maven project here. I'm pretty new to maven, Is there a better way to do it?